manuscript/elsarticle-num.bst

```diff
@@ -0,0 +1,1517 @@
+%% 
+%% Copyright 2007, 2008, 2009 Elsevier Ltd
```

Comment:
Yeah, it would have an error without it..
```
This is BibTeX, Version 0.99d (TeX Live 2015/Debian)
The top-level auxiliary file: ddca-litrev.aux
I couldn't open style file elsarticle-num.bst
---line 19 of file ddca-litrev.aux
 : \bibstyle{elsarticle-num
 :                         }
I'm skipping whatever remains of this command
I found no style file---while reading file ddca-litrev.aux
(There were 2 error messages)
Makefile:8: recipe for target 'all-via-pdf' failed
make: *** [all-via-pdf] Error 2
```
If it is only my machine then I can remove it!